components/uniset/src/props.rs

```diff
@@ -715,6 +715,7 @@ where
         Script::Yezidi => key::SCRIPT_YEZIDI_V1,
         Script::Yi => key::SCRIPT_YI_V1,
         Script::ZanabazarSquare => key::SCRIPT_ZANABAZAR_SQUARE_V1,
+        _ => return Err(UnicodeSetError::UnknownScriptId(enum_val.0)),
```

Comment:
Thought: This just means that you can't proactively request a particular UnicodeSet for a script that wasn't known at build time.  That is smelly, but I'll accept it.  If you really want to access the data for a new script from an old version of ICU4X, you should be able to get it from the CodePointTrie API.

Reply:
1. Proactive requests for currently unknown scripts are the long tail of the long tail, I think. There's not a lot of demand for Vithquki.
2. This restriction is driven by the way we represent our resource keys: you can't proactively request an unknown script because we don't have any way to retrieve it. If we refactored that, we could also rewrite this function to provide the data. The API doesn't have to change; Script can already represent unknown scripts. If there's demand, we can support it.